Shared/Views/CampaignList.swift

```diff
@@ -184,6 +216,33 @@ struct CampaignList: View {
                         .padding(.bottom)
                     }
                     
+                    if headToHeads.count > 0 {
+                        Button(action: {
+                            withAnimation {
+                                showHeadToHeads.toggle()
+                            }
+                        }) {
+                            HStack {
+                                Text("Head to Head")
+                                    .font(.title2)
+                                    .fontWeight(.bold)
+                                    .fullWidth()
+                                Spacer()
+                                Image(systemName: showHeadToHeads ? "chevron.down" : "chevron.right")
```

Comment:
FWIW, if you show the same chevron but rotate it 90 degrees based on whether it's showing head-to-heads or not, I believe the animation would look better